rest-api/core-rest/src/main/java/uk/gov/gchq/gaffer/rest/service/v2/example/DefaultExamplesFactory.java

```diff
@@ -391,6 +395,25 @@ public GenerateElements generateElements() {
         return op;
     }
 
+    @Override
+    public OperationChain<Iterable<? extends Element>> sort(){
```

Comment:
Please could you update this sort method so it returns: Sort

Then the example would just be:

```java
new Sort.Builder()
    .comparators(new ElementPropertyComparator.Builder()
            .groups("entity", "edge")
            .property("count")
            .reverse(true)
            .build())
    .resultLimit(20)
    .deduplicate(true)
    .build()
```

Reply:
Sure! Thanks!

Reply:
@p013570 Changes done please check